independent-publisher-connectors/snowflake/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1270 @@
+                    "type": "string",
+                    "example": "/api/v2/statements/GUID value"
+                  },
+                  "resultSetMetaData": {
+                    "type": "object",
+                    "properties": {
+                      "format": {
+                        "type": "string",
+                        "enum": [
+                          "jsonv2"
+                        ],
+                        "description": "For v2 endpoints the only possible value for this field is jsonv2.",
+                        "example": "jsonv2"
+                      },
+                      "rowType": {
+                        "type": "array",
+                        "minItems": 1,
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "name": {
+                              "type": "string"
+                            },
+                            "type": {
+                              "type": "string"
+                            },
+                            "nullable": {
+                              "type": "boolean"
+                            }
+                          },
+                          "additionalProperties": true
+                        }
+                      }
+                    }
+                  },
+                  "partitionInfo": {
+                    "description": "Partition information",
+                    "type": "array",
+                    "minItems": 0,
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "rowCount": {
+                          "type": "integer",
+                          "format": "int64",
+                          "description": "Number of rows in the partition.",
+                          "minimum": 0,
+                          "example": 1324
+                        },
+                        "compressedSize": {
+                          "type": "integer",
+                          "format": "int64",
+                          "description": "the partition size before the decompression. This may or may not be present in the partitionInfo. Uncompressed size would always be there.",
+                          "minimum": 0,
+                          "example": 37436824
+                        },
+                        "uncompressedSize": {
+                          "type": "integer",
+                          "format": "int64",
+                          "description": "the partition size after the decompression",
+                          "minimum": 0,
+                          "example": 1343787384
+                        }
+                      }
+                    }
+                  },
+                  "nullable": {
+                    "description": "false if null is replaced with a string 'null' otherwise false",

```

Comment:
@rekodus here in you description should the last work be true instead of false? if yes please fix and submit an update.
I want to notify that we have paused all our connectors deployment due to a deployment freeze until January 3rd. We will resume deployment in January you can submit your update then :-).

Please let me know if you have any questions.

Thank you so much.